_includes/login.md

```diff
@@ -1,5 +1,7 @@
 To sign into Quay.io, execute the `docker login quay.io` command:
 
+Note: If you go to your account settings you can create an encrypted password for more cli security.
```

Comment:
Can we link to the account settings page? Also I'd get rid of the "cli" and just have "more security".